documentation/translations/coordinating.rst

```diff
@@ -3,79 +3,214 @@ Coordinating
+
+Discussions about translations occur on the Python Docs Discord
+`#translations channel <https://discord.gg/h3qDwgyzga>`_, `translation
+mailing list <translation_ml_>`_, and the
+`translations category <trans_disc_>`_ of the Python Discourse.
+
+For administrative issues, ping ``@python/editorial-board``.
+
 
 Starting a new translation
 ==========================
 
-First subscribe to the `translation mailing list <translation_ml_>`_,
-and introduce yourself and the translation you're starting.
+Coordination is not a one-off task, it is a long-term commitment. Before
+you start your translation, carefully consider if you will be able to commit
+to this.
+Every coordinator should be familiar with translating: see :ref:`translating`.
+
+The following sections will guide you through setting up your translation.
+If you have any questions or need help, ask in one of the
+:ref:`help channels <translation-help>`.
+
+
+Announcement
+------------
+
+Post an announcement introducing yourself and the translation you're
+starting on `Discourse <trans_disc>`_. Also join the other communication
+channels, if possible.
+
+
+Coordination team
+-----------------
+
+Each translation team will decide on the number of coordinators.
+While initially one person is fine, we recommend expanding to having two or
+three coordinators. You can find more on choosing coordinators in the FAQ below.
+
+.. XXX move stuff from the FAQ?
+
 
-Then you can bootstrap your new translation by using the `cookiecutter
+Translation team
+----------------
+
+.. figure:: translator-workload.svg
+   :class: invert-in-dark-mode
+   :align: center
+   :alt: An exaggerated chart showing that individual translator workload
+         decreases with the amount of translators.
+
+
+Gather people to help you translate. You can't do it alone.
+Update :ref:`this table <translation-coordinators>` via a PR to make
+your translation easier to find. In the entry you can also include links to
+guides or other resources for translators.
+
+
+Repository
+----------
+
+To start your translation create a GitHub repository, under any
+account, with the correct Git hierarchy and folder structure. This can be done
+in several ways, and will dictate the translation process you use.
+
+Each translation is assigned an appropriate lowercase
+`IETF language tag <https://datatracker.ietf.org/doc/html/rfc5646.html>`_.
+The tag may have an optional region subtag, joined with a dash.
+For example, ``pt`` (Portuguese) or ``pt-br`` (Brazilian Portuguese).
+The repository name is then: ``python-docs-TAG``
+
+The name of each branch should be the Python version it holds translations
+for, for example, ``3.14``. The files should be structured like the source files
+in `CPython/Doc <https://github.com/python/cpython/tree/main/Doc>`_.
+A correctly set up repository looks like this:
+`python-docs-pl <https://github.com/python/python-docs-pl/>`_
+
+
+Cookiecutter/bootstrapper
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I'd make this and Translations platform / Transifex sections have the same level as _Repository_